specs/credentialmanagement/index.src.html

```diff
@@ -747,20 +747,24 @@ <h3 id="request-credential">
       Let <var>settings</var> be the <a>incumbent settings object</a>.
     </li>
     <li>
-      If <var>settings</var> does not have a <a>responsible document</a>, then
-      reject <var>promise</var> with a <code>SecurityError</code>
-      <strong>and</strong> terminate this algorithm.
-    </li>
-    <li>
-      If <var>settings</var>' <a>responsible document</a> is not the <a>active
-      document</a> in the <a>top-level browsing context</a>
-      If <var>environment</var> is not a <a>top-level browsing context</a>,
+      If any of the following conditions are met:
+      <ul>
+        <li>
+          <var>settings</var> does not have a <a>responsible document</a>
+        </li>
+        <li>
+          <var>settings</var>' <a>responsible document</a> is not the
+          <a>active document</a> in the <a>top-level browsing context</a>
+        </li>
+        <li>
+          <var>settings</var> is not a <a>privileged context</a>
+        </li>
+        <li>
+          <var>environment</var> is not a <a>top-level browsing context</a>
```

Comment:
And, I know you just added in a previous patch, but please drop the `environment` definition as well. We don't need it, as the "incumbent settings object" is the settings object we're referring to with `environment`.